css/main.sass

```diff
@@ -118,6 +118,8 @@ header.shrink
     transition: height 0.2s
     height: 80px
     position: fixed
+    #social
```

Comment:
It is a little confusing that we have both an ID and a class that are named the same thing but mean fundamentally different things—perhaps could we just reference `&:first-child` under `.social`?